.gitignore

```diff
@@ -1,4 +1,5 @@
-data/data/
+/data/clean/
+/data/raw/
```

Comment:
I don't think these 2 lines are necessary, try removing them and seeing if git tracks the data